admin/src/User/index.tsx

```diff
@@ -47,3 +53,35 @@ export const UserCreate = (props: any) => (
     </SimpleForm>
   </Create>
 );
+
+export const UserShow = (props: any) => (
+  <Show {...props}>
+    <SimpleShowLayout>
+      <TextField source="id" />
+      <TextField source="email" />
+      <UserGame />
```

Comment:
You could probably have just used an [referencemanyfield](https://marmelab.com/react-admin/Fields.html#referencemanyfield) !?